docs/src/components/demo/Demo.tsx

```diff
@@ -24,7 +24,11 @@ export function Demo({ className, defaultOpen = false, title, ...props }: DemoPr
   return (
     <BaseDemo.Root className={clsx('DemoRoot', className)} {...props}>
       <ErrorBoundary FallbackComponent={DemoErrorFallback}>
-        <BaseDemo.Playground aria-label="Component demo" className="DemoPlayground" />
+        <BaseDemo.Playground
+          suppressHydrationWarning
```

Comment:
What's going on during hydration?

Reply:
Oh weird, I used to have hydration warnings here related to some control ID attributes in the rendered demos, but not seeing this anymore. Had shut it off while I was working on something else and forgot to get back to this, but it's not giving me the hydration warnings now 🤔 

I'll keep an eye on it, for now removed `suppressHydrationWarning`